exercises/concept/guessing-game/.docs/introduction.md

```diff
@@ -0,0 +1,42 @@
+# Introduction
+
+While `if/else` expressions can be used to execute conditional logic, Haskell also has a  more powerful way to execute conditional logic: pattern matching. With pattern matching, a value can be tested against one or more _patterns_. An example of such a pattern is the _constant pattern_, which matches a value against a constant (e.g. `1` or `"hello"`).
+
+When defining functions, you can define separate function bodies for different patterns. This leads to clean code that is simple and readable. You can pattern match on any data type — numbers, characters, lists, tuples, etc. 
+
+For example, a trivial function that checks if the number supplied to it is a `1` or `0` could be expressed like this:
+
+```haskell
+describe :: Int -> String
+describe 0 = "Zero"
+describe 1 = "One"
```

Comment:
do not forget to use `closerToZero` here as well. What should be the difference between guessing-game's introduction.md and the concept's introduction.md? If they are the same, shouldn't we symlink them, so that we don't forget to make changes in all three different places?